app/helpers/email_helper.rb

```diff
@@ -1,5 +1,9 @@
 module EmailHelper
   def current_subdomain
-    Apartment::Tenant.current == "public" ? "" : Apartment::Tenant.current 
+    if Hours.single_tenant_mode?
+      ""
+    else
+      Apartment::Tenant.current == "public" ? "" : Apartment::Tenant.current
+    end
```

Comment:
In single tenant mode we need to check if application uses domain if yes then we need to return the subdomain, if not the we return ''. So is possible that it won't be one line condition syntax .